content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+
+### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<br />
+<a href="https://docs.sourcegraph.com/cloud"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Scale confidently with a dedicated and secure instance of Sourcegraph Cloud
+After almost a decade of developing self-hosted Sourcegraph, we’ve gained the trust of some of the most sophisticated companies and development teams in the world, allowing us to work with their most sensitive IP: their code. Today—after years of building trust and engineering rigor—we are excited to move confidently to the Cloud with a highly secure and scalable solution.
+
+Dedicated, single-tenant Sourcegraph Cloud instances are now generally available, and the best way for new teams to use Sourcegraph. You can read more in our announcement blog here[/blog/enterprise-cloud].
+
+<a href="https://docs.sourcegraph.com/cloud" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+  
+<br />
+<a href="https://docs.sourcegraph.com/cloud"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### SOC 2 Type II certification for Sourcegraph Cloud
+Sourcegraph continues our commitment to the security of your most valuable IP—your code—through our security operations, compliance measures, and best practices. With our recent SOC 2 Type II attestation for Soucegraph Cloud, we met the high standards of the third-party audit, and you can be confident that Sourcegraph controls for the security and availability of your sensitive data. 
+
+<a href="https://security.sourcegraph.com/" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### 📈 Advanced admin capabilities
+<br />
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
+We know that upgrading Sourcegraph can be a time-intensive process, especially if you fall a few versions behind. Sourcegraph 4.0 supports multi-version upgrades so you can upgrade to 4.0 directly from version 3.20 or higher.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Multi-version%20upgrades%20final',
+    mp4: 'blog/release-post/4.0/Multi-version%20upgrades%20final'
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Upgrade to 4.0 directly from version 3.20 or higher with multi-version upgrades"
+/>
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<a href="https://docs.sourcegraph.com/admin/analytics"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Understand usage and quantify the value of using Sourcegraph with in-product analytics
```

Comment:
@ryphil @macraigyou please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.

Reply:
LGTM! Thank you!

Reply:
TY for reviewing!